src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/ReviewApprenticeshipUpdatesRequestToViewModelMapper.cs

```diff
@@ -34,6 +39,13 @@ public async Task<ReviewApprenticeshipUpdatesViewModel> Map(ReviewApprenticeship
             {
                 var update = updates.ApprenticeshipUpdates.First();
 
+                if (!string.IsNullOrWhiteSpace(update.TrainingName))
+                {
+                    var apiRequest = new CheckReviewApprenticeshipCourseRequest(source.ProviderId, source.ApprenticeshipId);
```

Comment:
We could use this ticket as an opportunity to migrate some of the inner api calls above into the new pattern. I would suggest that we make this request unconditional, and add the apprenticeshipTask result properties to the outer api result (so, firstname, lastname, employername, not sure why we need providername). Then, we could remove the apprenticeshipTask altogether and save on one api call from the UI. 

Better still would be to remove the updatesTask as well, but it might be worth leaving this for another time due to the time involved.

I also think that `CheckReviewApprenticeshipCourseRequest` and related objects could be renamed to better reflect the whole page.